web/deeamon128/html/signup-form/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/47ce19f1c7.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+  <title>Signup Form</title>
+</head>
+<body>
+  <form action="#" class="form">
+    <div class="icon"><i class="fas fa-times"></i></div>
+    <h1>Join our community of photography lovers from all over the world</h1>
+    <div class="inputs">
```

Comment:
for clarity here, you can choose a better class name as an example, "email-wrapper"

Reply:
instead of which class you mean?